protoc-gen-orion/internal/testprotos/go.mod

```diff
@@ -0,0 +1,82 @@
+module github.com/carousell/Orion/protoc-gen-orion/internal/testprotos
```

Comment:
Introduce a `internal/testprotos` sub-module for isolating the `github.com/carousell/Orion` dependency.
With this file, protoc-gen-orion wouldn't need to depend on `github.com/carousell/Orion`.